src/client/js/otp/modules/planner/PlannerModule.js

```diff
@@ -21,9 +21,7 @@ otp.modules.planner.defaultQueryParams = {
     date                            : moment().format(otp.config.locale.time.date_format),
     arriveBy                        : false,
     mode                            : "TRANSIT,WALK",
-    maxWalkDistance                 : 1609.34, // 1 mile
```

Comment:
Per my comment https://github.com/CUTR-at-USF/usf-mobullity/issues/98#issuecomment-149992750, I think these two variables should stay - they still exist upstream.